hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/FileSizeCountTask.java

```diff
@@ -91,11 +92,17 @@ public Pair<String, Boolean> reprocess(OMMetadataManager omMetadataManager) {
         reprocessBucketLayout(BucketLayout.LEGACY, omMetadataManager,
             fileSizeCountMap);
     if (!statusFSO && !statusOBS) {
-      return new ImmutablePair<>(getTaskName(), false);
+      return new TaskResult.Builder()
+          .setTaskName(getTaskName())
+          .setTaskSuccess(false)
+          .build();
```

Comment:
If we find ourselves repeating the same TaskResult creation code frequently, it can be beneficial to extract it into a helper method. 

```suggestion
      if (!statusFSO && !statusOBS) {
        return buildTaskResult(false);
      }
```

New method added would be for this class :- 
```
private TaskResult buildTaskResult(boolean success) {
  return new TaskResult.Builder()
      .setTaskName(getTaskName())
      .setTaskSuccess(success)
      .build();
}
```

Not a major deal but would create simplicity and a reduce the code. We could do the same for other classes as well.